pydle/features/tls.py

```diff
@@ -58,6 +58,7 @@ async def _connect(self, hostname, port, reconnect=False, password=None, encodin
 
     async def whois(self, nickname):
         info = await super().whois(nickname)
+        if info is None: return info
```

Comment:
(style) please put the return statement on a second line.
```suggestion
        if info is None: 
            return info
```